gramps/gui/dbman.py

```diff
@@ -1027,6 +1185,227 @@ def __drag_data_received(self, widget, context, xpos, ypos, selection,
+        dlg.add_buttons(
+            Gtk.STOCK_OK, Gtk.ResponseType.OK
+        )
+        self.__targetname.set_tooltip_text(_("The name of your database"))
+        self.window.vbox.pack_start(self.__targetname, False, True, 0)
+        label7 = Gtk.Label(label="")
+        self.window.vbox.pack_start(label7, False, True, 0)
+        load_db = Gtk.Button(label=_("Load a gedcom/gramps file?"))
+        load_db.set_tooltip_text(_("Click here if you want to load a gedcom/"
+                                   "gramps file."))
+        load_db.connect("clicked", self.load_database_selection)
+        load_db.set_halign(Gtk.Align.START)
+        self.window.vbox.pack_start(load_db, False, True, 0)
+        self.__target1 = Gtk.Entry()
+        self.__target1.set_text('')
+        self.__target1.set_tooltip_text(_("If you want to load a gedcom/"
+                                          "gramps file, select it here."))
+        self.window.vbox.pack_start(self.__target1, False, True, 0)
+        self.window.set_default_size(600, 300)
+        self.window.connect('response', self.start)
+        self.window.show_all()
+        self.show()
+
+    def start(self, obj, resp):
+        """
+        Do we start the first gramps session ?
+        """
+        if resp == Gtk.ResponseType.OK:
+            name = self.__targetname.get_text()
+            file_to_load = self.__target1.get_text()
+            if file_to_load == "":
+                try:
+                    file_to_load = tempfile.mktemp(suffix=".gramps")
+                    ftl = open(file_to_load, "wb")
+                except IOError as msg:
+                    ErrorDialog(_("Default gedcom/gramps file for novice"),
+                                _("Could not create %s: %s") % (file_to_load,
+                                                                msg),
+                                parent=self.uistate.window)
+                except:
+                    ErrorDialog(_("Default gedcom/gramps file for novice"),
+                                _("Could not create %s") % file_to_load,
+                                parent=self.uistate.window)
+
+                ftl = open(file_to_load, "w", encoding='utf-8')
+                xml = FIRST_TIME_DEFAULT_FILE.format(
+                    me=_("Me"),
+                    partner1=_("Father/partner1"),
+                    partner2=_("Mother/partner2"),
+                    surnm1=_("Surname"),
+                    surnm2=_("Mother/partner2 Surname"),
+                    surnm3=_("Wife Surname"),
+                    child1=_("Child"),
+                    brother=_("Brother"),
+                    wife=_("Wife"),
+                    place1=_("Country"),
+                    place2=_("County"),
+                    place3=_("Place"),
+                    place4=_("Second place")
+                    )
+                ftl.write(xml)
+                ftl.close()
+                os.unlink(file_to_load)
```

Comment:
This is not any better; now it is deleted before you get to load it on line 1377.

Reply:
Yes. I think the nex patch will solve this.